tz_abbr_infos.go

```diff
@@ -520,7 +520,7 @@ var tzAbbrInfos = map[string][]*TzAbbreviationInfo{
 			isDST:       false,
 			name:        "Pacific Standard Time",
 			offset:      -25200,
```

Comment:
```suggestion
			offset:      -28800,
```

Could you please also fix it?

Reply:
Ah, my bad, totally missed it.